src/librustc_middle/mir/visit.rs

```diff
@@ -134,6 +134,13 @@ macro_rules! make_mir_visitor {
                 self.super_ascribe_user_ty(place, variance, user_ty, location);
             }
 
+            fn visit_coverage(&mut self,
+                              coverage: & $($mutability)? Coverage,
+                              _context: PlaceContext,
```

Comment:
I don't think we want/need `PlaceContext` here. That's used when there is a `Place` being visited which we don't have.

Reply:
done/removed